scene/gui/tab_container.cpp

```diff
@@ -998,6 +1030,8 @@ void TabContainer::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "drag_to_rearrange_enabled"), "set_drag_to_rearrange_enabled", "get_drag_to_rearrange_enabled");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "tabs_rearrange_group"), "set_tabs_rearrange_group", "get_tabs_rearrange_group");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "use_hidden_tabs_for_min_size"), "set_use_hidden_tabs_for_min_size", "get_use_hidden_tabs_for_min_size");
+	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "input_rollover"), "set_input_rollover", "get_input_rollover");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "tab_bar_focus", PROPERTY_HINT_ENUM, "None,Click,All"), "set_tab_bar_focus", "get_tab_bar_focus");
```

Comment:
```suggestion
	ADD_PROPERTY(PropertyInfo(Variant::INT, "tab_bar_focus", PROPERTY_HINT_ENUM, "None,Click,All"), "set_tab_bar_focus", "get_tab_bar_focus_mode");
```
Simple "focus" is a bit misleading.

Reply:
Fixed